docs/design/features/lsra-detail.md

```diff
@@ -393,109 +384,110 @@ the `LsraBlockInfo`.
+            normally have a predecessor block that has already
+            been allocated. This facility is exercised by the 0x200
+            (`LSRA_BLOCK_BOUNDARY_LAYOUT`) setting of `COMPlus_JitStressRegs`.
 
-        -   Build `Interval`s with annotations
+-   At the end of a block, for any exposed uses that do not have downstream
+    `RefPosition`s (e.g. variables that are live across the backedge, so there is no
+    last use), a `RefTypeExposedUse` is created.
 
-            -   Ref positions (called use positions by [[1]](#[1]) at each
-                def or use, and for internal registers.
+During this phase, preferences are set:
 
-            -   Add cross-interval preferences
+-   Cross-interval preferences are expressed via the `relatedInterval` field of `Interval` 
 
-                -   This area has room for improvement [Issue #11463](https://github.com/dotnet/coreclr/issues/11463)
+    -   When a use is encountered, it is preferenced to the target `Interval` for the
+        node, if that is deemed to be profitable. During register selection, it tries to
+        find a register that will fit the requirements of that preferenced `Interval`.
+        Then, when the use `RefPosition` is assigned a register, the
+        register preference of the target `Interval` is updated.
+        This addresses the scenario when the lclVar has already
+        (at a previous definition) been assigned a register, and we want to try to use that
+        register again, as well as the case where it has yet to be assigned a register.
 
-            -   Determine whether to prefer callee-save registers. [Issue
-                \#7664](https://github.com/dotnet/coreclr/issues/7664) tracks the need to improve the heuristics for
-                when to prefer callee-save.
+        This area has room for improvement:
 
-            -   Add pseudo-uses to the `Interval` if live at loop backedge.
+        -   A specific case that could be improved is [Issue #25312](https://github.com/dotnet/coreclr/issues/25312)
+            which involves preferencing for HW intrinsics.
 
-            -   If the previous block is not a predecessor of the
-                current block, add DummyDefs for variables that are
-                live-in to the current block, but not live-out of the
-                previous block.
+        -   Issue [#22374](https://github.com/dotnet/coreclr/issues/22374) also has a pointer
+            to some methods that could benefit from improved preferencing.
+   
+    - Register preferences are set:
 
-                -   This is generally not required, as the block will
-                    normally have a predecessor block that has already
-                    been allocated.
+        - When the use or definition of a value must use a fixed register, due to instruction
+          or ABI constraints. In this case, the register may be added to the `registerPreferences`,
```

Comment:
It depends on whether it looks like it might conflict with the existing preferences. The heuristics for deciding this are in `mergeRegisterPreferences()` I'll add a note to that effect.